src/components/Footer/Footer.js

```diff
@@ -11,45 +11,69 @@ function Footer() {
     <>
       <div className={css.banner}>
         <img className={css.left_banner} alt="left_banner" src={left} />
-      </div>
-      <div className={css.banner}>
         <img className={css.right_banner} alt="right_banner" src={right} />
       </div>
+      
       <div className={css.second_position}>
         <div className={css.use_info}>
           <span className={css.fontUse}> 이용안내 </span>
```

Comment:
className 컨벤션에 맞게 수정 부탁드립니다!
```suggestion
          <span className={css.font_use}> 이용안내 </span>
```

Reply:
고쳤습니다.